packages/auth-providers/dbAuth/api/src/shared.ts

```diff
@@ -178,25 +190,36 @@ export const getSession = (
 // at once. Accepts the `event` argument from a Lambda function call and the
 // name of the dbAuth session cookie
 export const dbAuthSession = (
-  event: APIGatewayProxyEvent,
+  event: APIGatewayProxyEvent | Request,
   cookieNameOption: string | undefined
 ) => {
-  if (extractCookie(event)) {
+  const sessionCookie = extractCookie(event)
+  const bearerToken = extractEncryptedSessionFromHeader(event)
+
+  if (sessionCookie) {
+    // i.e. Browser making a request
     const [session, _csrfToken] = decryptSession(
-      getSession(extractCookie(event), cookieNameOption)
+      getSession(sessionCookie, cookieNameOption)
     )
+    return session
+  } else if (bearerToken) {
```

Comment:
@cannikin not sure about this one here. I don't think we need it right? This was for our first implementation of dbAuth ssr

Reply:
Yeah if you're just adding the cookie itself to the headers then we don't need this auth header bearer token check any more!

Reply:
Thanks, taken out. Will just be forwarding on the cookie in the SSR-Auth version